packages/slate-react/src/index.js

```diff
@@ -1,5 +1,6 @@
 
 import Editor from './components/editor'
+import cloneFragment from './utils/copy-fragment'
```

Comment:
This one got missed I think and is causing the tests to tail. Once those are passing I'm down to merge! Thanks @justinweiss!